app/channels/activity_channel.rb

```diff
@@ -43,21 +43,18 @@ def process_asset_group(strong_params)
     begin
       uuid_assets = Asset.where(uuid: asset_uuids).to_a
       barcode_assets = Asset.find_or_import_assets_with_barcodes(asset_barcodes)
-
-      # Maintaining existing behaviour: register previously unknown fluidx barcodes
-      fluidx_barcodes = asset_barcodes.select { |bc| TokenUtil.is_valid_fluidx_barcode?(bc) }
-      missing_barcodes = fluidx_barcodes - barcode_assets.map(&:barcode)
```

Comment:
We removed fluidx tube creation, but inadvertently left this code around, and also used it to identify the missing barcodes. This meant that we didn't detect missing non-fluidx barcodes.